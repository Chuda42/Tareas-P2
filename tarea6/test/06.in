# Implementar vecinos
vecinos 1
vecinos 2
vecinos 9
vecinos 10
hacerVecinos 5 2 25
vecinos 2
vecinos 5
hacerVecinos 5 9 59
vecinos 2
vecinos 5
vecinos 9
hacerVecinos 2 9 29
vecinos 1
vecinos 2
vecinos 5
vecinos 9
vecinos 10
hacerVecinos 5 1 100
hacerVecinos 5 3 100
hacerVecinos 4 5 100
hacerVecinos 6 5 100
hacerVecinos 5 7 100
hacerVecinos 5 8 100
hacerVecinos 5 10 100
vecinos 1
vecinos 2
vecinos 5
vecinos 9
vecinos 10
Fin
