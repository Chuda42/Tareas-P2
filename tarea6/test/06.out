1>#  Implementar vecinos.
2>
3>
4>
5>
6>Nuevos vecinos: 5 2 25.00.
7>5 
8>2 
9>Nuevos vecinos: 5 9 59.00.
10>5 
11>2 9 
12>5 
13>Nuevos vecinos: 2 9 29.00.
14>
15>5 9 
16>2 9 
17>2 5 
18>
19>Nuevos vecinos: 5 1 100.00.
20>Nuevos vecinos: 5 3 100.00.
21>Nuevos vecinos: 4 5 100.00.
22>Nuevos vecinos: 6 5 100.00.
23>Nuevos vecinos: 5 7 100.00.
24>Nuevos vecinos: 5 8 100.00.
25>Nuevos vecinos: 5 10 100.00.
26>5 
27>5 9 
28>1 2 3 4 6 7 8 9 10 
29>2 5 
30>5 
31>Fin.
